### ISSUE TITLE
Did anything changed in popover syntax since last release?

### ISSUE THREAD

**Issue by TheCodeExorcist** [thednp/bootstrap.native#365]
Hi again,
After a bit of struggle to build a UMD from the latest sources (after discovering that the docs are outdated, i followed what's indicated in #359 ), i now have a problem with popovers.
Popovers using data api are working well, however those who were declared through JS no more work. I previously declared my popovers using this exact syntax (the syntax used in bootstrap native docs for v3):
`new Popover(inputs[i],{
      container: "body",
      placement: "top",
      trigger: "manual",
      title: "Attention!",
      content: "Verrouillage majuscule actif"
    });`

It worked like a charm when i was using the latest available release, but now firefox developer throws me this error:
`ReferenceError: Popover is not defined`
The error location corresponds to "Popover" in the previously mentioned code.

I think i'm probably issuing a lack of documentation following a syntax change. Am i doing something wrong?

**Comment by thednp** [comment]
If you're doing a custom build make sure you have everything you need, but before that, make sure it works with the default build. Do some console.log before, after and in between.

**Comment by thednp** [comment]
Your options values are pretty off the line, you should check the docs (the options part of the documentation is the same basically).

**Comment by TheCodeExorcist** [comment]
After checking the docs, it seems i used some twbs options. I removed them but it's still not working. The error is not about a wrong parameter or something like that, it clearly says that the Popover class does not exist by the time i try to make a new popover object out of it.
However this exact same script works pretty well with the latest standard bsn release.
Actually, it is not really a custom build: i just tried to do a UMD build with what's inside the current master branch with the stock index.js file.
`npm run custom INPUTFILE:src/index.js,OUTPUTFILE:dist/bootstrap-native-custom.js,MIN:false,FORMAT:umd`

Except this, the build runs fine and so far all the functionalities i have tested are working well. No errors occured during the build process. As i mentionned, popovers initialized through DATA API are working well, i only can't initialize popovers through JS. 
The javascript files are loading in the correct order (BSN then my custom JS), i don't understand why this class isn't defined at this point.

**Comment by TheCodeExorcist** [comment]
I found the solution (by instinct, nothing scientific here)!
I needed to declare my popover this way:
`new BSN.Popover(inputs[i],{ container: "body", placement: "top", title: "Attention!", content: "Verrouillage majuscule actif" });`
instead of:
`new Popover(inputs[i],{ container: "body", placement: "top", title: "Attention!", content: "Verrouillage majuscule actif" });`

Probably something has changed since the last release concerning this. I don't know if this behavior is intended.
